.github/workflows/main.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        verify_flag: ["", "--vac", "--cex"]
+        verify_flag: ["", "--vac", "--cex --horn-explicit-sp0=true"]
```

Comment:
we do not need `explicit-sp0` for all jobs. Only for some. Please add it only when necessary. It under approximates some behaviours. I am still unsure whether using it all the time will not miss some bugs.